tests/unit/pages/Login.spec.js

```diff
@@ -107,10 +110,24 @@ describe("Login.vue", () => {
         await flushPromises();
 
         expect(requestOtp).toHaveBeenCalled();
-        expect(wrapper.vm.resentOtp).toBe(true);
+        expect(startResendOTPTimer).toHaveBeenCalled();
+        expect(wrapper.vm.resendOTPTimer).toBe(60);
+        expect(wrapper.vm.isResendOTPEnabled).toBe(false);
```

Comment:
you'll likely have to advance jest timer by 60 seconds so that the resend otp button becomes active so that the trigger at line 108 works - currently that trigger is not working.

Reply:
if the trigger was not working then how come resendOTPTimer value was being set?
I didn't understand this.